.loco/config/mysql-common/my.cnf.php

```diff
@@ -37,6 +37,7 @@ function matchVer($pat) {return (bool) preg_match($pat, ver());}
 <?php if (matchVer('/Ver 5.6/')) { ?>
 innodb_large_prefix = 1
 <?php } ?>
+<?php if (!matchVer('/Ver 8.0/')) { ?>innodb_file_format = Barracuda<?php } ?>
```

Comment:
@totten this was to fix some Min test failures I have no idea how to actually apply this so found the relevant files and manually fixed them on the server

Reply:
Good q. Easiest ways to apply this:

* For a local environment (where one uses `loco run`), just add the `-f` / `--force` option (i.e. `loco run -f`).
* For CI environment (where there are several systemd services), use `nix/bin/reset-ci.sh`.

In both cases, it'll re-generate the concrete config files and start the services.

(There are also more fine-tuned options like `loco init -f mysql`.)